docs/api/content.md

```diff
@@ -40,204 +25,117 @@ view.
 Content, along with many other Ionic components, can be customized to modify its padding, margin, and more using the global styles provided in the [CSS Utilities](/docs/layout/css-utilities) or by individually styling it using CSS and the available [CSS Custom Properties](#css-custom-properties).
 
 
-## Fixed Content
+## Basic Usage
 
-In order to place elements outside of the scrollable area, `slot="fixed"` can be added to the element. This will absolutely position the element placing it in the top left. In order to place the element in a different position, style it using [top, right, bottom, and left](https://developer.mozilla.org/en-US/docs/Web/CSS/position).
+import Basic from '@site/static/usage/content/basic/index.md';
 
-## Interfaces
+<Basic />
 
-### ScrollBaseDetail
 
-```typescript
-interface ScrollBaseDetail {
-  isScrolling: boolean;
-}
-```
+## Header & Footer
 
-### ScrollDetail
+Content can be the only top-level component in a page, or it can be used alongside a [header](./header), [footer](./footer), or both. When used with a header or footer, it will adjust its size to fill the remaining height.
 
-```typescript
-interface ScrollDetail extends GestureDetail, ScrollBaseDetail {
-  scrollTop: number;
-  scrollLeft: number;
-}
-```
-
-### ScrollBaseCustomEvent 
+import HeaderFooter from '@site/static/usage/content/header-footer/index.md';
 
-While not required, this interface can be used in place of the `CustomEvent` interface for stronger typing on the `ionScrollStart` and `ionScrollEnd` events.
-
-```typescript
-interface ScrollBaseCustomEvent extends CustomEvent {
-  detail: ScrollBaseDetail;
-  target: HTMLIonContentElement;
-}
-```
-
-### ScrollCustomEvent 
-
-While not required, this interface can be used in place of the `CustomEvent` interface for stronger typing on the `ionScroll` event.
-
-```typescript
-interface ScrollCustomEvent extends ScrollBaseCustomEvent {
-  detail: ScrollDetail;
-}
-```
+<HeaderFooter />
 
 
+## Fullscreen Content
 
+By default, content fills the space between a [header](./header) and [footer](./footer) but does not go behind them. In certain cases, it may be desired to have the content scroll behind the header and footer, such as when the `translucent` property is set on either of them. This can be achieved by setting the `fullscreen` property on the content to `true`.
 
-## Usage
+import Fullscreen from '@site/static/usage/content/fullscreen/index.md';
 
-<Tabs groupId="framework" defaultValue="angular" values={[{ value: 'angular', label: 'Angular' }, { value: 'javascript', label: 'Javascript' }, { value: 'react', label: 'React' }, { value: 'stencil', label: 'Stencil' }, { value: 'vue', label: 'Vue' }]}>
+<Fullscreen />
```

Comment:
Good catch! I actually went looking and the content doesn't even go behind the header at all in Material Design, so it looks like the `fullscreen` isn't actually working with `md` at all. I will update!

Reply:
Okay I updated the demo so that it sets `opacity` on the toolbar instead of using `translucent` on the header/footer which allows us to showcase `fullscreen` on `md` mode as well. The content background was added to really show off that it goes behind them but let me know if you think this is too much, it's pretty bold.

As for switching between modes, you will have to click "Reset Demo" when switching between them to see the fullscreen working. I created this PR to fix this: https://github.com/ionic-team/ionic-docs/pull/2623

Reply:
I think the background could be a little brighter to make the text easier to read, but otherwise this looks great 👍 Thanks for digging in!